src/components/PostDetails/PostDetails.tsx

```diff
@@ -1,45 +1,108 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
```

Comment:
I guess there is no need to import the whole React library. Try to not import it if its possible